react-popup/my-app/src/App.tsx

```diff
@@ -20,7 +27,8 @@ function App() {
         <Popup
           open={isOpen}
           onClose={() => setIsOpen(false)}
-          positionTo={myRef.current}></Popup>
+          positionTo={myRef.current}
+          children={menuItems}></Popup>
```

Comment:
The best way to write this is to put `children` as actual children of `Popup`, like this:
```
<Popup
          open={isOpen}
          onClose={() => setIsOpen(false)}
          positionTo={myRef.current}>
    <ul className="list-none shadow-2xl border">
          {menuItems.map((i) => <li>{i}</li>)}
    </ul>
</Popup>
```

This way, you can put any JSX you want in the popup. It doesn't have to be a list of strings. It could be buttons, images, etc., etc.